src/riot-rs-macros/src/task.rs

```diff
@@ -111,7 +111,7 @@ pub fn task(args: TokenStream, item: TokenStream) -> TokenStream {
             #task_function
         }
     } else {
-        let pool_size = attrs.pool_size;
+        let pool_size = attrs.pool_size.unwrap_or_else(|| syn::parse_quote! { 1 });
```

Comment:
Could we add the info about this default value to the docs?

Reply:
I don't think that's clear from the current documentation:
> - `pool_size`: (*optional*) set the maximum number of concurrent tasks that can be spawned for the function.

Based on these docs I would assume that per default there is no maximum number of tasks that can be spawned for the function.